### PR TITLE
Update cluster-autoscaler version for metrics

### PR DESCRIPTION
**Pull request by tuommaki**
In order to be able to scrape metrics of cluster-autoscaler with prometheus, it
must expose a service that prometheus-config-controller scrapes. Therefore the
version of chart must be updated.

### CODE REVIEW

File: service/controller/aws/v10/key/key.go [hunk 1]
```diff
@@ -23,7 +23,7 @@ func ChartSpecs() []key.ChartSpec {
 		},
 		{
 			AppName:           "cluster-autoscaler",
-			ChannelName:       "0-1-stable",
+			ChannelName:       "0-2-stable",
```
Comment (xh3b4sd) [inline service/controller/aws/v10/key/key.go]: Can you point me to where this is coming from?

Comment (tuommaki) [reply service/controller/aws/v10/key/key.go]: https://github.com/giantswarm/kubernetes-cluster-autoscaler/blob/master/helm/kubernetes-cluster-autoscaler-chart/Chart.yaml#L4 given that I understood it correctly how this works. Or did I misunderstand this?

Comment (MarcelMue) [reply service/controller/aws/v10/key/key.go]: Current way of doing things in guest cluster charts:
You can push multiple times into the same channel if the change is minor ( this should work now and not cause any problems).
If your change is bigger then you should update the `minor` like tuomas has done here.

We don't have this written down anywhere right now and it isn't intuitive IMO. This is changing with the current WIP stories, so we deal with it until they get done.

**Comment by tuommaki** [comment]
Ok. E2e on cluster-operator is not healthy atm. so I'm going with this.

**Pull request merged**